lib/config.js

```diff
@@ -7,7 +7,10 @@ var config = require('bedrock').config;
 var path = require('path');
 require('bedrock-validation');
 
-config['key-http'] = {};
+// var basePath = '/keys';
+// config['key-http'] = {};
+// config['key-http'].routes = {};
+// config['key-http'].routes.basePath = basePath;
```

Comment:
I don't know why basePath is needed in bedrock-key-http if it's in bedrock-key.  What is the usefulness of aliasing the var to a different namespace?  I don't think we need to leave the commented code in the file.  Is there a policy for adding an empty config or just omit it?